### ISSUE TITLE
Square endpoint

### ISSUE THREAD

**Issue by tristancamejo** [twisttaan/placevie#2]
Have a square endpoint with a size parameter instead of a width and height. so a size of 500 would make a width and height of 500x500. I’m pretty sure we could use a next redirect or some other next feature to implement this, other wise we should refactor the current generate endpoint to be a function called generatePlaceholder and call it in the square and normal generate endpoint.

**Comment by GodderE2D** [comment]
Why do we need a separate endpoint for this? Just check if the `size` parameter is there, and if so, use `size * size` for width and height? Otherwise, fall back to width and height values.

**Comment by tristancamejo** [comment]
Fair point, cheeze requested me to make this project and after showing off the first version he requested for a “square endpoint”.

**Comment by GodderE2D** [comment]
> Fair point, cheeze requested me to make this project and after showing off the first version he requested for a “square endpoint”.

your first issue is that you honoured cheeze's request

**Comment by twisttaan** [comment]
> > Fair point, cheeze requested me to make this project and after showing off the first version he requested for a “square endpoint”.
> 
> your first issue is that you honoured cheeze's request

💀
